collector/mongod/collections_status.go

```diff
@@ -74,15 +80,24 @@ func (collStatList *CollectionStatList) Export(ch chan<- prometheus.Metric) {
 		collectionObjectCount.With(ls).Set(float64(member.Count))
 		collectionAvgObjSize.With(ls).Set(float64(member.AvgObjSize))
 		collectionStorageSize.With(ls).Set(float64(member.StorageSize))
-		collectionIndexes.With(ls).Set(float64(member.Indexes))
+		collectionIndexes.With(ls).Set(float64(len(member.IndexSizes)))
```

Comment:
@martinhoefling Could you describe what was wrong here?

Reply:
I had to map `bson:"indexSizes,omitempty"` in `CollectionStatus` struct above to a different type to make it available for a per index size map. Therefore, i need to figure out the number of total indexes slightly different.